platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/BundleNameCondition.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.condition;
+
+import java.util.Dictionary;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Pattern;
+import org.osgi.framework.Bundle;
+import org.osgi.service.condpermadmin.Condition;
+import org.osgi.service.condpermadmin.ConditionInfo;
+
+public class BundleNameCondition implements Condition {
+
+  private static final ConcurrentHashMap<String, Boolean> DECISION_MAP =
+      new ConcurrentHashMap<>(100000);
+
+  private static final Pattern REGEX = Pattern.compile("/");
+
+  private Bundle bundle;
+  private String[] args;
+
+  public BundleNameCondition(Bundle bundle, ConditionInfo conditionInfo) {
+    this.bundle = bundle;
+    args = conditionInfo.getArgs();
+  }
+
+  @Override
+  public boolean isPostponed() {
+    return false;
+  }
+
+  @Override
+  public boolean isSatisfied() {
+    if (args.length == 0) {
+      return false;
+    }
+    String bundleName = bundle.getHeaders().get("Bundle-SymbolicName");
```

Comment:
❓ How much do we care that symbolic name isn't unique? We do a pretty good job of following a naming convention that limits collisions, but there's nothing preventing them.

Reply:
It is all we have so it is going to have to be good enough.

Reply:
You don't have access to the maven coordinates from the bundle.